airflow-core/src/airflow/executors/local_executor.py

```diff
@@ -138,7 +138,7 @@ class LocalExecutor(BaseExecutor):
 
     It uses the multiprocessing Python library and queues to parallelize the execution of tasks.
 
-    :param parallelism: how many parallel processes are run in the executor
+    :param parallelism: how many parallel processes are run in the executor, must be > 0
```

Comment:
Same as above:


```suggestion
    :param parallelism: how many parallel processes are run in the executor, must be >= 0
```

Reply:
Looking at https://github.com/apache/airflow/blob/main/airflow-core/src/airflow/executors/base_executor.py#L177-L178 the value 0 is rejected since PR https://github.com/apache/airflow/pull/48218 and also according to devlist thread https://lists.apache.org/thread/jlroywp6tlcrgdsx28shl1nkll8bpvbt

Reply:
Ah! My mistake, looks good :)